spec/services/update_duplicate_matches_spec.rb

```diff
@@ -22,6 +22,22 @@
   end
 
   describe '#save!' do
+    context 'when existing fraud match' do
+      before do
+        described_class.new.save!
+        # e.g The support team unblocks the candidate
```

Comment:
I added this comment to explain the part of the process as an example. Please do let me know if you want to remove this.